src/Microsoft.Restier.WebApi/Model/ApiModelBuilder.cs

```diff
@@ -11,16 +11,17 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.OData.Edm;
 using Microsoft.OData.Edm.Library;
+using Microsoft.Restier.Core;
 using Microsoft.Restier.Core.Model;
 using Microsoft.Restier.Core.Query;
 
-namespace Microsoft.Restier.Core.Conventions
+namespace Microsoft.Restier.WebApi.Model
 {
     /// <summary>
     /// A convention-based API model builder that extends a model, maps between
     /// the model space and the object space, and expands a query expression.
     /// </summary>
-    internal class ConventionBasedApiModelBuilder
+    internal class ApiModelBuilder
```

Comment:
This is used to builder some model for OData, we have some discussion, we do not think other protocol can reuse this code as we do not think there are two protocol shares the same model. This is the reason we move to WebApi.

If user  wants to use OData with RESTier, then we expect user always have WebApi publisher, if user uses other protocol, we do not expect they will need this service.